.gitignore

```diff
@@ -38,3 +38,5 @@ build
 lerna-debug.log
 yarn-error.log
 *.orig
+yarn.lock
+package-lock.json
```

Comment:
We actually want to upload `yarn.lock` if it's present

```suggestion
package-lock.json
```

Reply:
@icazevedo 

Corrections made.